CalculateCalendarLogic/CalculateCalendarLogic.swift

```diff
@@ -164,7 +164,10 @@ public struct CalculateCalendarLogic {
      * ※3. [Swift] 関数における引数/戻り値とタプルの関係：http://dev.classmethod.jp/smartphone/swift-function-tupsle/
      *
      */
-    public func judgeJapaneseHoliday(year: Int, month: Int, day: Int) -> Bool {
+    public func judgeJapaneseHoliday(year: Int,
```

Comment:
__[SwiftLint]__ Function should have complexity 10 or less: currently complexity equals 50

<sub>posted by Sider</sub>